### PR TITLE
Added initial bug fixes found when testing

### PR DESCRIPTION
**Pull request by isaacrivriv**

### CODE REVIEW

File: express_cut/express_service/serializers.py
Comment (TheParodicts) [inline express_cut/express_service/serializers.py]: It looks like the logic that's trying to be implemented here may benefit in readability from using the enumerate(list) function. Here's a link to a description of it: https://note.nkmk.me/en/python-enumerate-start/#:~:text=Tags%3A%20Python%2C%20List-,enumerate()%20in%20Python,built%2Din%20function%20enumerate()%20.

File: frontend/src/App.js
Comment (TheParodicts) [inline frontend/src/App.js:105]: What is the purpose of adding this logic here?

File: frontend/src/components/stylists/activeAppointmentView.jsx
Comment (TheParodicts) [inline frontend/src/components/stylists/activeAppointmentView.jsx:45]: What is the reason for this change?

File: frontend/src/components/stylists/scheduleManagementView.jsx
Comment (TheParodicts) [inline frontend/src/components/stylists/scheduleManagementView.jsx:227]: Both of these statements should be equivalent. If there is no responseData, responseDAta.id should evaluate to false.

File: frontend/src/components/stylists/scheduleManagementView.jsx
Comment (TheParodicts) [inline frontend/src/components/stylists/scheduleManagementView.jsx:68]: I don't think this breaks anything but it is standard in JS to use 3 = signs for comparitive operations.

File: frontend/src/components/stylists/stylistSidebarRoutes.js
Comment (TheParodicts) [inline frontend/src/components/stylists/stylistSidebarRoutes.js:11]: This change duplicates code. what is the purpose of it?

**Review by TheParodicts — CHANGES_REQUESTED** [review]
There are a number of small changes that affect small, but core, functionalities of the frontend, and I would like to discuss them before signing off on this merge. It seems to introduce some redundant code, go against some programming style conventions, and change some fundamental logic.

**Review by TheParodicts — APPROVED** [review]
Tested the code, and it seems to work. If this is the way you guys want it, it gets my approval.

File: frontend/src/components/stylists/scheduleManagementView.jsx
Comment (milton-pagan) [reply frontend/src/components/stylists/scheduleManagementView.jsx:68]: What @TheParodicts  said is correct. Not using === could potentially break something.

**Review by milton-pagan — APPROVED** [review]

File: frontend/src/components/stylists/scheduleManagementView.jsx
Comment (isaacrivriv) [reply frontend/src/components/stylists/scheduleManagementView.jsx:68]: I understand that. Not sure where but the type of objects were giving incorrect results. Thats why the change happened. If you guys want to get to the core issue and fix it definetely give it a chance but its a fix in the meantime

**Review by milton-pagan — APPROVED** [review]

File: frontend/src/components/stylists/stylistSidebarRoutes.js
Comment (isaacrivriv) [reply frontend/src/components/stylists/stylistSidebarRoutes.js:11]: Not sure were it duplicates code. I removed the active reservation view from the Manager since it wasnt working correctly giving a lot of 404s it was trying to get the current reservation from the manager which doesnt work as well. The only change was removing part of the view. Could have been done differently by perhaps removing it from the array but its a fix in the meantime if you guys can find a better way to work it

File: frontend/src/components/stylists/scheduleManagementView.jsx
Comment (isaacrivriv) [reply frontend/src/components/stylists/scheduleManagementView.jsx:227]: Should but failed once I tried it. Therefore the fix. It seems responseData was undefined which getting the id from undefined broke things. Again temporary fix to get stuff out the door. If you can find the core issue and fix it definetely do

File: frontend/src/components/stylists/activeAppointmentView.jsx
Comment (isaacrivriv) [reply frontend/src/components/stylists/activeAppointmentView.jsx:45]: ActiveAppointment was not working with Manager. It was trying to get the current appointments for the manager getting a lot of 404s. Didnt see a way to switch it to others stylists like the reservation view does so it seems this only refers to stylists not managers so made the change to hide it if a manager logs in

File: frontend/src/App.js
Comment (isaacrivriv) [reply frontend/src/App.js:105]: Refer to comment below

File: express_cut/express_service/serializers.py
Comment (isaacrivriv) [reply express_cut/express_service/serializers.py]: Ill look into it and try to make the change

Comment (isaacrivriv) [reply express_cut/express_service/serializers.py]: Ok made the changes to use enumerate

**Pull request merged**
